src/DynamoCore/Graph/Workspaces/NodesToCodeExtensions.cs

```diff
@@ -104,7 +104,6 @@ internal static class NodesToCodeExtensions
 
                 var codeBlockNode = new CodeBlockNodeModel(
                     code,
-                    System.Guid.NewGuid(),
```

Comment:
??

Reply:
This happens inside the other constructor.